### DISCUSSION TITLE
AdGuard Home Error When Test Upstream and Dns List

### DISCUSSION THREAD

**Discussion by faizakbar18** [AdguardTeam/AdGuardHome#3173] [Q&A]
### Issue Details

* **Version of AdGuard Home server:**
  * v0.106.3
* **How did you install AdGuard Home:**
  * Docker
* **CPU architecture:**
  * ARM64
* **Operating system and version:**
  * 18.04.5 LTS (Bionic Beaver)

### Expected Behavior
AdGuard Home Run Normally

### Actual Behavior
AdGuard Home like no have internet connection.
Test Upstream result 

> Server "https://dns10.quad9.net/dns-query": could not be used, please check that you've written it correctly

Default DNS Blocklist show 0 rulest count, and get error when adding new blocklist (See Screensoots)
### Screenshots

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/24320262/119239800-99c00c80-bb75-11eb-97af-4e10787d7ebb.png)

</details>

### Additional Information

[Log File](https://pastebin.com/99w0XYE4)

Command that i Used to run:
```
docker run --name adguardhome \
    -v /data/adguard/work:/opt/adguardhome/work \
    -v /data/adguard/conf:/opt/adguardhome/conf \
    -p 53:53/tcp -p 53:53/udp \
    -p 3000:3000/tcp \
    -d adguard/adguardhome
```
UI interface change to port 3000, so after installing using same port

i already using this AdGuard Home from another Pi without docker

**Comment by ameshkov** [comment]
Probably, Quad9 is unaccessible from your place: `i/o timeout, read udp 172.17.0.4:50144->149.112.112.10:53: i/o timeout `

Try using different DNS servers in both upstreams and bootstrap sections.

**Reply by faizakbar18** [reply]
All dns get same error

**Reply by ameshkov** [reply]
Note that you need to change bootstrap servers as well

**Reply by faizakbar18** [reply]
but its work normally without docker
